client/package.json

```diff
@@ -12,6 +12,7 @@
     "font-awesome": "^4.7.0",
     "razorpay-checkout": "^1.0.4",
     "react": "^17.0.2",
+    "react-avatar": "^4.0.0",
```

Comment:
why are we using react-avatar for? can't we simply use avatar from bootstrap?
is this really necessary?

Reply:
its not needed

Reply:
I will remove it